src/overview.md

```diff
@@ -240,6 +240,24 @@ More information:
 * [`webdev` tool](/tools/webdev)
 * [Web deployment tips](/web/deployment)
 
+#### The Dart Runtime {#runtime}
```

Comment:
It seems a little weird to have this section under "Dart: The platforms", since the preceding sections are "Dart Native" and "Dart Web".

What do you think of having a new, second-level "Dart: The runtime" section? If that seems wrong to you, maybe we could change the intro to tie into the platforms. Something like "Regardless of which platform you use or how you compile your code, executing the code requires a Dart runtime."

Reply:
Also, I'm dying to fix the white space ([semantic line breaks](https://github.com/dart-lang/site-shared/blob/master/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks). I could do it in a followup PR, or I could do it now (while restraining myself from changing anything else).

Do you have a preference?

Reply:
I directly edited dart-compile.md (as described in the commit) since it didn't need many changes. Could you please check my work, @mit-mit?

Reply:
| Also, I'm dying to fix the white space

Sorry, all other repos in the org use automatically wrapped paragraphs at 80 cols, so I keep forgetting about that :-(

Reply:
Edits LGTM!

Reply:
I think the question about moving up these new "Dart: The runtime" details to a higher level still remains. Taking a look, I agree it's a bit strange to be with the platforms, so perhaps a new second-level heading like you mentioned would be good.

Reply:
I think it's tolerable: the runtime itself is part of the platforms, and the platforms content here really is more of an architectural overview.

Reply:
Not quite, but if we change the intro sentence to "Regardless of which platform you use or how you compile your code, executing the code requires a Dart runtime" then I'm OK with it.

Reply:
> @kwalrath I think we can merge this?

I'll change that intro sentence and (in a separate commit) fix the formatting. If those are OK by you, then I'll merge this.

Reply:
SGTM!